jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/sso/bearer/ConfirmUserCodeLoginDialog.kt

```diff
@@ -68,6 +73,27 @@ class ConfirmUserCodeLoginDialog(
         super.doCancelAction()
         AwsTelemetry.loginWithBrowser(project = null, Result.Cancelled, credentialType)
     }
+
+    override fun createActions(): Array<Action> {
+        val defaultButtons = arrayOf(cancelAction, okAction)
+        return if (ApplicationInfo.getInstance().build.productCode != "GW") {
+            defaultButtons
+        } else {
+            val activeConnection = CodeCatalystCredentialManager.getInstance(null).connection()
+            if (activeConnection != null) {
+                arrayOf(SignInWithDifferentAccountButton(), cancelAction, okAction)
+            } else {
+                defaultButtons
+            }
+        }
+    }
+
+    private inner class SignInWithDifferentAccountButton : DialogWrapperAction(message("aws.sso.signing.device.code.diff.account")) {
+        override fun doAction(e: ActionEvent?) {
+            close(CLOSE_EXIT_CODE)
```

Comment:
we probably need to cancel the `pollForToken` call instead of letting it continue in the background

Reply:
Cancelled the progress bar